.gitlab-ci.yml

```diff
@@ -58,93 +58,40 @@ after_script:
   artifacts:
     name: "$CI_JOB_NAME"
     paths:
-      - _install_ci
-      - config/Makefile
-      - config/coq_config.py
-      - test-suite/misc/universes/all_stdlib.v
+      - _build
```

Comment:
Doesn't this contain more than we install? We want to test install so that seems wrong.

Reply:
Install isn't tested by the test suite, it's tested by the various 3rd party developments.